packages/libs/react-ui/src/components/Notification/NotificationContainer.tsx

```diff
@@ -1,10 +1,12 @@
+// eslint-disable-next-line @typescript-eslint/consistent-type-imports
```

Comment:
Since I'm only using the colorVariants const in an interface, ESLint turns it into a type import while it's not so I added this line